File: lsclient.client/src/app/models/TrucksPayload.ts
export interface TrucksPayload {
    truckID: string;
    truckNumber: string;
    model: string;
    truckTypeID: string;
    companyID: string;
    isActive: boolean;
    isTruckAvilableForBooking: boolean;
    driverID: string;
    chasisNo?: string;
    brand?: string;
    engineCapacity?: string;
    fuelType?: string;
    cabinType?: string;
    category?: string;
    drive?: string;
  }
  

File: lsclient.Server/Models/DataPayloads/TrucksPayload.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace lsclient.Server.Models.DataPayloads
{
    public class TrucksPayload
    {
            public string? TruckID { get; set; }
            public string? TruckNumber { get; set; }
            public string? Model { get; set; }
            public string? CompanyID { get; set; }
            public string? DriverID { get; set; }

            public string? TruckTypeID { get; set; }
            public bool? IsActive { get; set; }
           public bool? IsTruckAvilableForBooking { get; set; }

           public string? chasisNo { get; set; }
           public string? Brand { get; set; }
           public string? EngineCapacity { get; set; }
           public string? FuelType { get; set; }
           public string? CabinType { get; set; }
           public string? Category { get; set; }
           public string? Drive { get; set; }



    }
}
